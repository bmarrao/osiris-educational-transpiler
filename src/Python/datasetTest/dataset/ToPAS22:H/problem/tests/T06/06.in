5
6 10
MHgXz qI QXr jP jmAI Zhu
jP MHgXz
QXr qI
jP qI
MHgXz Zhu
qI MHgXz
jP Zhu
Zhu qI
QXr Zhu
jmAI jP
Zhu jmAI
6 4
u CElL cUkLi dX Qbr WIhC
Qbr CElL
CElL u
dX u
u WIhC
6 2
F gR yvbng HNJuD FRB eNdWP
yvbng gR
F gR
6 2
se gYTl XzwD aap K aff
K gYTl
se XzwD
6 1
Baex QT yfH fDV VNh aHTQ
VNh yfH
