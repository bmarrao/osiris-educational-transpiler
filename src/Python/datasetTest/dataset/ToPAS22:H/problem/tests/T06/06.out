Impossivel
Possivel: 240
Possivel: 432
Possivel: 480
Possivel: 576
